index.html

```diff
@@ -1942,7 +1942,7 @@ <h4 id="maturity-levels">6.1.2 Maturity Levels</h4>
            be restored to Recommendation Status. See also clause 10 of the licensing requirements for W3C Recommendations in
           <a href="https://www.w3.org/Consortium/Patent-Policy#sec-Requirements">section 5</a> of the
           <a href="https://www.w3.org/Consortium/Patent-Policy" data-ref="PUB33">W3C Patent Policy</a>.</dd>
-        <dt id="WGNote">Working Group Note, Interest Group Note (NOTE) </dt>
+        <dt id="WGNote">Working Group Note, Interest Group Note (<abbr title="Internet Relay Chat">NOTE</abbr>) </dt>
```

Comment:
`title="Internet Relay Chat"` looks like a copy+paste mistake.

If we want to use abbr, I think we should change the whole line to:
```
<dt id="WGNote"><abbr title="Working Group Note, Interest Group Note">NOTE</abbr>) </dt>
```
and do the same for every other entry in the list. (REC, CR, WD...).

At the same time, leaving this as-is seems fine to me.

What I think we should *really* do is convert the whole document to bikeshed, and use its facilities for defining and cross referencing terms.